python/python/bystro/ancestry/tests/test_listener.py

```diff
@@ -1,100 +1,38 @@
-"""Test ancestry listener."""
-
-from pathlib import Path
-from unittest.mock import patch
-
-import pytest
-
-from bystro.ancestry.ancestry_types import AncestrySubmission
 from bystro.ancestry.listener import (
     AncestryJobData,
-    completed_msg_fn,
     handler_fn_factory,
-    submit_msg_fn,
 )
-from bystro.ancestry.tests.test_inference import ANCESTRY_MODEL, FAKE_GENOTYPES
+from bystro.ancestry.tests.test_inference import (
+    ANCESTRY_MODEL,
+    FAKE_GENOTYPES,
+    FAKE_GENOTYPES_DOSAGE_MATRIX,
+)
 from bystro.beanstalkd.messages import ProgressMessage
 from bystro.beanstalkd.worker import ProgressPublisher
 
-FAKE_VCF_DIR = Path("my_fake_vcf_dir")
 
+import pyarrow.feather as feather # type: ignore
 
 handler_fn = handler_fn_factory(ANCESTRY_MODEL)
 
 
-def test_handler_fn_happy_path():
-    progress_message = ProgressMessage(submissionID="my_submission_id")
-    publisher = ProgressPublisher(
-        host="127.0.0.1", port=1234, queue="my_queue", message=progress_message
-    )
-    ancestry_submission = AncestrySubmission("foo.vcf")
-    ancestry_job_data = AncestryJobData(
-        submissionID="my_submission_id2", ancestry_submission=ancestry_submission
-    )
-    with patch("bystro.ancestry.listener._load_vcf", return_value=FAKE_GENOTYPES) as _mock:
-        ancestry_response = handler_fn(publisher, ancestry_job_data)
-    assert ancestry_submission.vcf_path == ancestry_response.vcf_path
-
-
-def test_submit_msg_fn_happy_path():
```

Comment:
Consider adding this back

Reply:
Done, added tests for submit_msg_fn and completed_msg_fn